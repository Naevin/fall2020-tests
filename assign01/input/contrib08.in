a :- (2 + 3 * c) / b;
b :- c + 5;
c :- 7 * (d + 1) / 44;
d :- 3 ^ (5 + 4 ^ 2);
(d * 2 + 1) + a;
