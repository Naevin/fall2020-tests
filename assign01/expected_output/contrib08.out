Result: 20920706409
